cookbook/docs/index.md

```diff
@@ -0,0 +1,377 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(getting_started_index)=
+
+# Introduction to Flyte
+
+[Flytesnacks Tags](_tags/tagsindex)
+
+Flyte is a workflow orchestrator that seamlessly unifies ETL/ELT,
+machine learning, and analytics stacks for building robust and reliable
+applications.
+
+This quickstart guide provides an overview of how to get Flyte up and running
+on your local machine.
+
+## Installation
+
+```{admonition} Prerequisites
+:class: important
+
+[Install Docker](https://docs.docker.com/get-docker/) and that the Docker daemon
+is running.
+```
+
+First install [flytekit](https://pypi.org/project/flytekit/), Flyte's Python SDK.
+
+```{prompt} bash $
+pip install flytekit
+```
+
+Then install [flytectl](https://docs.flyte.org/projects/flytectl/en/latest/),
+which the command-line interface for interacting with a Flyte backend.
+
+````{tabbed} OSX
+
+```{prompt} bash $
+brew install flyteorg/homebrew-tap/flytectl
+```
+
+````
+
+````{tabbed} Other Operating systems
+
+```{prompt} bash $
+curl -sL https://ctl.flyte.org/install | sudo bash -s -- -b /usr/local/bin
+```
+
+````
+
+## Creating a Workflow
+
+The first workflow we'll create is a simple model training workflow that consists
+of three steps:
+
+1. 🍷 Gets the classic [wine dataset](https://scikit-learn.org/stable/datasets/toy_dataset.html#wine-recognition-dataset)
+   using [sklearn](https://scikit-learn.org/stable/).
+2. 📊 Processes the data that simplifies the 3-class prediction problem into a
+   binary classification problem by collapsing class labels `1` and `2` into
```

Comment:
Are we collapsing or are we nullifying class 2?

Reply:
collapsing. basically 0 stays 0, 1 and 2 are collapsed to 1